src/sankeyDiagram.ts

```diff
@@ -911,29 +897,42 @@ export class SankeyDiagram implements IVisual {
     private static getTooltipForNode(
         valueFormatter: IValueFormatter,
         nodeName: string,
-        nodeWeight: number,
+        nodeInputWeight: number,
+        nodeOutputWeight: number,
         localizationManager: ILocalizationManager,
-        nodeDisplayName?: string,
         valueDisplayName?: string,
     ): VisualTooltipDataItem[] {
 
-        let formattedNodeWeight: string;
+        let formattedNodeInputWeight: string;
+        let formattedNodeOutputWeight: string;
 
         if (valueFormatter && valueFormatter.format) {
-            formattedNodeWeight = valueFormatter.format(nodeWeight);
+            formattedNodeInputWeight = valueFormatter.format(nodeInputWeight);
+            formattedNodeOutputWeight = valueFormatter.format(nodeOutputWeight);
         } else {
-            formattedNodeWeight = nodeWeight.toString();
+            formattedNodeInputWeight = nodeInputWeight.toString();
+            formattedNodeOutputWeight = nodeOutputWeight.toString();
         }
 
-        return [
+        const tooltips: VisualTooltipDataItem[] = [
             {
                 displayName: localizationManager.getDisplayName("Visual_TooltipDisplayName"),
                 value: nodeName
-            }, {
-                displayName: valueDisplayName || SankeyDiagram.RoleNames.values,
-                value: formattedNodeWeight
             }
         ];
+
+        if (valueDisplayName) {
+            tooltips.push({
+                displayName: SankeyDiagram.InputTooltipValue + valueDisplayName || SankeyDiagram.RoleNames.values,
```

Comment:
is '+' a good idea of string concatenation?

Reply:
fixed and localized 09b4985a6c4338db9d540ec2b4cc86a342279eea